release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1055,6 +1055,25 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* Currently, Operator Lifecycle Manager (OLM) displays pod security admission (PSA) warnings for catalog sources installed outside of the `openshift-marketplace` namespace. 
++
+Workaround: Cluster administrators can remove the warning by setting the level of the audit mode to `baseline`, as shown in the following example:
++
+.Example `manifests/<namespace>.yaml` file
++
+[source,yaml]
```

Comment:
Suggest adding the `resource` name here so that the user clearly know add this `label` to `namespace` resource. Such as
```yaml
apiVersion: v1
kind: Namespace
metadata:
...
```

Reply:
Addressed